config/config.go

```diff
@@ -31,6 +31,7 @@ type Config struct {
 
 	// Hot Feed
 	RunHotFeedRoutine bool
+	HotFeedMediaRequired bool
```

Comment:
```suggestion
	RunHotFeedRoutine      bool
	HotFeedMediaRequired bool
```
Go-fmt is going to want this aligned